requirements_to_watch.txt

```diff
@@ -1,4 +1,2 @@
-# These requirements prevented an upgrade to Django 1.10.
-django-coverage-plugin==1.5.0
```

Comment:
Why is `django-coverage-plugin` being removed?

Reply:
I removed it from requirements_to_watch.txt because it's already in local.txt and test.txt as an actual requirement (see #1325).

Reply:
Ok, missed that one. Good spot!